roles/openshift_control_plane/templates/master.yaml.v1.j2

```diff
@@ -2,11 +2,6 @@ kind: MasterConfig
 apiVersion: v1
 admissionConfig:
   pluginConfig:{{ openshift.master.admission_plugin_config | default(None) | lib_utils_to_padded_yaml(level=2) }}
-    PodPreset:
```

Comment:
We need to use yedit to remove this from existing clusters, which is what the bug seems to indicate.

We don't re-use templates during upgrades.

Reply:
Its already being removed during upgrade: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_control_plane/tasks/upgrade.yml#L98, this would happen for both minor and major upgrades